client/src/components/ProductItems/ProductItems.module.scss

```
@@ -0,0 +1,42 @@
+@import "../../variable.scss";
+
+.ProductItems {
+	* {
+		margin-left: 2%;
+		margin-right: 2%;
+	}
+	.heartIcon {
+		width: 3.5rem;
+		padding: 7px;
+	}
+
+	div {
+		text-align: center;
+	}
+
+	.itemImages {
+		height: 9rem;
+		max-width: 100%;
+		border-radius: 20px;
+	}
+	.productName {
+		font-size: $fz-medium-1;
+		color: $fc-5;
+		font-weight: 300;
```

Hier hat mazlum ne variable für
```suggestion
		font-weight: $fw-light;
```
bzw. @supermazlum muss sie noch hinzufügen